app/main.py

```diff
@@ -16,4 +16,31 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+dictionary_variables = {
+    "mutable": [list, dict, set],
+    "immutable": [int, float, bool, str, tuple]
+}
+
+sorted_variables = {
+    "mutable": [],
+    "immutable": []
+}
+
+for key, values in dictionary_variables.items():
+    for value in values:
+        if value == type(lucky_number):
+            sorted_variables[key].append(lucky_number)
+        if value == type(pi):
+            sorted_variables[key].append(pi)
+        if value == type(one_is_a_prime_number):
+            sorted_variables[key].append(one_is_a_prime_number)
+        if value == type(name):
+            sorted_variables[key].append(name)
+        if value == type(profile_info):
+            sorted_variables[key].append(profile_info)
+        if value == type(my_favourite_films):
+            sorted_variables[key].append(my_favourite_films)
+        if value == type(marks):
+            sorted_variables[key].append(marks)
+        if value == type(collection_of_coins):
+            sorted_variables[key].append(collection_of_coins)
```

Comment:
You just need to populate the `sorted_variables` dict here with variables from above, no additional logic is needed.

Reply:
ok, i fixed it)))